# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1363205107 14400
#      Wed Mar 13 16:05:07 2013 -0400
# Node ID d39342a19fb85bf73c7aa7caa2e2d3950f531fb3
# Parent  23c5f90f92cd44b5aaf944be2d8c5f78b2ea6e15
* ov-float.cc (SCALAR_MAPPER): Use gnulib::floorf.

diff --git a/libinterp/octave-value/ov-float.cc b/libinterp/octave-value/ov-float.cc
--- a/libinterp/octave-value/ov-float.cc
+++ b/libinterp/octave-value/ov-float.cc
@@ -301,17 +301,17 @@ octave_float_scalar::map (unary_mapper_t
       SCALAR_MAPPER (lgamma, rc_lgamma);
       SCALAR_MAPPER (cbrt, ::cbrtf);
       SCALAR_MAPPER (ceil, ::ceilf);
       SCALAR_MAPPER (cos, ::cosf);
       SCALAR_MAPPER (cosh, ::coshf);
       SCALAR_MAPPER (exp, ::expf);
       SCALAR_MAPPER (expm1, ::expm1f);
       SCALAR_MAPPER (fix, ::fix);
-      SCALAR_MAPPER (floor, ::floorf);
+      SCALAR_MAPPER (floor, gnulib::floorf);
       SCALAR_MAPPER (log, rc_log);
       SCALAR_MAPPER (log2, rc_log2);
       SCALAR_MAPPER (log10, rc_log10);
       SCALAR_MAPPER (log1p, rc_log1p);
       SCALAR_MAPPER (round, xround);
       SCALAR_MAPPER (roundb, xroundb);
       SCALAR_MAPPER (signum, ::signum);
       SCALAR_MAPPER (sin, ::sinf);
